src/main/webapp/WEB-INF/view/addUserBox.jsp

```diff
@@ -0,0 +1,18 @@
+<!-- Add user modal -->
+<div id="addUserBox">
+
+  <!-- modal content -->
+  <div id="addUserContent">
+
+    <span class="close">&times;</span>
+    <p> Add people to this conversation! </p>
+    <div id ="searchDiv">
```

Comment:
Might be a good idea to use a class instead of an id now that we have the search bar in multiple places.

Reply:
Good idea.